src/sentry/options/defaults.py

```diff
@@ -1455,3 +1455,10 @@
     default=[],
     flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
 )
+
+# flag for turning on/off the options_automator_slack_webhook.
+register(
+    "options_automator_slack_webhook",
+    default=False,
+    flags=FLAG_MODIFIABLE_BOOL | FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
```

Comment:
Please remove 
- `FLAG_MODIFIABLE_BOOL` . You do not need to be modified by the admin tool
- `FLAG_PRIORITIZE_DISK`. There is no reason to set this via setting.